invenio_formatter/templates/invenio_formatter/macros/meta.html

```diff
@@ -73,3 +73,27 @@
     <meta name="citation_pdf_url" content="{{pdf_url}}" />
     {%- endif %}
 {%- endmacro %}
+
+{# Generate a badge in svg format. #}
+{%- macro badge(title, value, color='#007ec6') %}
+    <svg xmlns="http://www.w3.org/2000/svg">
+        <linearGradient id="b" x2="0" y2="100%">
+            <stop offset="0" stop-color="#bbb" stop-opacity=".1"/>
+            <stop offset="1" stop-opacity=".1"/>
+        </linearGradient>
+        <mask id="a">
+            <rect width="86" height="20" rx="3" fill="#fff"/>
+        </mask>
+        <g mask="url(#a)">
+            <path fill="#555" d="M0 0h31v20H0z"/>
+            <path fill="{{ color }}" d="M31 0h55v20H31z"/>
+            <path fill="url(#b)" d="M0 0h86v20H0z"/>
+        </g>
+        <g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11">
+            <text x="15.5" y="15" fill="#010101" fill-opacity=".3">{{ title }}</text>
```

Comment:
Ok, I have seen that these values for the X and Y are fixed but they should depend on the size of the title and value.